src/components/Vuetable.vue

```diff
@@ -377,14 +377,13 @@ export default {
       this.httpOptions['params'] = this.getAllQueryParams()
 
       if (this.httpMethod.toLowerCase() == 'post'){
```

Comment:
@moh3n65 Can you please fork from `develop` branch and re-submit the PR again include the fix suggested by @lgullestrup? This would help me integrate the PR into the next release.